system-tests/lib/system-tests.ts

```
@@ -613,7 +613,12 @@ const systemTests = {
       const s = this.servers
 
       if (s) {
-        await Bluebird.map(s, stopServer)
+        try {
+          await Bluebird.map(s, stopServer)
+        } catch (err) {
+          console.error('Error stopping server', err)
+          throw err
```

do we need to throw the error or reject the promise?

---

this is an async function cb to afterEach; throwing will reject that async fn's returned promise. This was a little bit of debugging to try and figure out why one of the system tests could not pass when run with `.only`, to no avail

---

Does the reject actually get called directly then?

---

There's no reject in this function, it's not a manual promise - the diff was confusing because of the leftover addition to the bluebird fn args. This is in the server stop logic, not the server start logic